crates/matrix-sdk/src/room/mod.rs

```diff
@@ -2941,7 +2941,7 @@ impl Room {
     pub async fn set_unread_flag(&self, unread: bool) -> Result<()> {
         let user_id = self.client.user_id().ok_or(Error::AuthenticationRequired)?;
 
-        let content = MarkedUnreadEventContent::new(unread);
+        let content = UnstableMarkedUnreadEventContent::from(MarkedUnreadEventContent::new(unread));
```

Comment:
To keep the same behavior as before, we need to use the type with the unstable prefix. A migration might need to be written to switch to the stable version.